docs/permissions.md

```diff
@@ -0,0 +1,112 @@
+- Once the coordinator reveals its awareness of keys, devices can disclose information about held shares:
+  - For AWARE keys: disclose index and share image
+  - For UNAWARE keys: provide a t-of-n key share and key fingerprint that progresses the coordinator towards becoming AWARE
+    - Coordinator must visit `t` devices to become AWARE
+    - Key names may be provided alongside shares
+
+**Confirmation**: Device _could_ REQUIRE CONFIRMATION before informing the coordinator of its held key fingerprints & t-of-n recovery shares.
+
+### Actions
+
+| Action                 | Access Requirement for Coordinator  | On-Device Confirmation |
+| ---------------------- | ----------------------------------- | ---------------------- |
+| KeyGen Start           | AWARE of any key (or no keys exist) | No                     |
+| KeyGen Finish          | AWARE (or no keys exist^^)          | Yes                    |
+| Share Public Nonces    | AWARE or UNAWARE (limited)          | No                     |
+| Signing                | AWARE of relevant key               | Yes                    |
+| Display Backup         | AWARE of relevant key               | Yes                    |
+| Check Backup           | AWARE                               | Yes (keyboard input)   |
+| Restore Backup         | AWARE or UNAWARE                    | Yes                    |
+| Device Data Wipe       | UNAWARE                             | Yes (CPU countdown)    |
+| Factory Firmware Reset | AWARE of ALL keys                   | Yes (CPU countdown?)   |
+| Delete Shares          | AWARE of relevant key               | Yes (long hold)        |
+| Update Firmware        | AWARE of any key                    | Yes                    |
+| Rename Device          | AWARE of any key                    | Yes                    |
+| Device Configuration   | AWARE of ALL keys                   | Yes                    |
+
+## Action Details
+
+### Nonces
+
+Sharing nonces during coordinator recovery enables two-round signing.
+Though we do not want to allow unknown coordinators to request heaps of nonces under different identities to crash the devices.
+Is this limit on nonces during key restoration enough to prevent storage overflow?
+Perhaps the device could store any pending new-coordinator nonces in a separate location, only storing one at a time.
+
+### Device Data Wipe
+
+If someone evil finds your device, they could destroy or steal it. Allowing someone UNAWARE to wipe the data off the device doesn't give them much besides hard to notice deletion. For this reason the data wipe should delete all device data such that it is noticed.
+
+This should require a long CPU countdown.
+Perhaps the timer to delete could scale with the number of shares held.
+
+## Device Configuration
+
+In the future, we might want a way to semi-permanently apply settings to a device, such as:
+
+- Disabled Actions
+- Speak to anyone mode
+- CPU timer multiplier
```

Comment:
I'd guess that configuration is mostly to do with keys and not global. Other than name/appearance settings.